src/central-backup.rst

```diff
@@ -56,6 +56,15 @@ Setting up backups
 
   You can verify your Google Drive usage `on the Drive storage page <https://drive.google.com/settings/storage>`_. You may want to periodically remove older backups to free up space.
 
+.. _central-backup-immediate:
+
+Performing an immediate backup
+------------------------------
+
+It is possible to immediately download a backup of your database to your own computer. As of Central v1.1, you will still need to have managed backups configured. Once you do, you will see a button :guilabel:`Download Backup Now` next to the :guilabel:`Terminate` button near the top.
```

Comment:
I think it'd be helpful to mention that a backup can be triggered without a configuration if going through the API. A link to [the API docs](https://github.com/getodk/central-backend/blob/master/docs/api.md#using-an-ad-hoc-passphrase-post) would also be great. I think it's the kind of thing that is within reach of a fair number of folks using `curl`.